sdks/python/apache_beam/runners/worker/statesampler.pyx

```diff
@@ -150,11 +161,31 @@ cdef class StateSampler(object):
           # without the GIL.
           nsecs_ptr = &(<ScopedState>PyList_GET_ITEM(
               self.scoped_states_by_index, self.current_state_index)).nsecs
+          lull_nsecs_ptr = &(<ScopedState>PyList_GET_ITEM(
+              self.scoped_states_by_index, self.current_state_index)).lull_nsecs
```

Comment:
<!--thread_id:cc_143091727_t; commit:95830a027670626ce0f081991d56dac60c8ee8d1; resolved:1-->
<!--section:context-quote-->
> **charlesccychen** wrote:
> Yes, could you move `time_since_transition` to a local / class variable?  It would be good to get rid of that property on `ScopedState`, since it really isn't used anywhere else.
> 
> Please also see my comment below on whether we can get rid of the lull-reporting logic here and move it to the progress reporter.

<!--section:body-->
Resolved.